packages/data-service-generator/src/prepare-context.ts

```diff
@@ -26,7 +25,7 @@ export async function prepareContext(
   dSGResourceData: DSGResourceData,
   logger: ILogger,
   pluginInstallationPath?: string
-): Promise<Module[]> {
+): Promise<void> {
```

Comment:
Why did you change the type?

Reply:
To reflect the actual behaviour. It never returns Module[]